### PR TITLE
Test coverage check

### PR DESCRIPTION
**Pull request by tcrosado**
`npm install` should be run
then running `./node_modules/.bin/solidity-coverage` should return:
prtestcoverage

Later this should be added to Jenkins and provide a report on each pull request

### CODE REVIEW

**Review by DioPires — APPROVED** [review]
Everything run as expected

File: test/AppCoinsIAB.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ var AppCoinsIAB = artifacts.require("./AppCoinsIAB.sol");
 var AppCoins = artifacts.require("./AppCoins.sol");
```
Comment (tcrosado) [inline test/AppCoinsIAB.js]: Tests:
22 passing

Test coverage: 
84.09 %

Comment (tcrosado) [inline test/AppCoinsIAB.js]: #### Tests:
14 passing
##### 8 failing

#### Test coverage: 
79.55 %

Comment (tcrosado) [inline test/AppCoinsIAB.js]: ### Tests:
14 passing
#### 8 failing

### Test coverage: 
79.55 %

Comment (tcrosado) [inline test/AppCoinsIAB.js]: #### Tests:
14 passing
#### 8 failing

#### Test coverage: 
79.55 %

Comment (tcrosado) [inline test/AppCoinsIAB.js]: #### 8 failing
#### Tests:
14 passing

#### Test coverage: 
79.55 %

Comment (tcrosado) [inline test/AppCoinsIAB.js]: #### Tests:
#### 8 failing
14 passing

#### Test coverage: 
79.55 %

Comment (tcrosado) [inline test/AppCoinsIAB.js]: #### Tests:
22 passing

#### Test coverage: 
84.09 %

Comment (tcrosado) [inline test/AppCoinsIAB.js]: #### Tests:
'''diff + 22 passing
'''
#### Test coverage: 
84.09 %

Comment (tcrosado) [inline test/AppCoinsIAB.js]: #### Tests:
'''diff + 22 passing
'''
#### Test coverage: 
84.09 %

Comment (tcrosado) [inline test/AppCoinsIAB.js]: #### Tests:
```diff + 22 passing
```
#### Test coverage: 
84.09 %

Comment (tcrosado) [inline test/AppCoinsIAB.js]: #### Tests:
```diff 
 + 22 passing
```
#### Test coverage: 
84.09 %

Comment (tcrosado) [inline test/AppCoinsIAB.js]: #### Tests:
```diff 
 - 8 failing
``````diff 
 + 14 passing
```
#### Test coverage: 
79.55 %

Comment (tcrosado) [inline test/AppCoinsIAB.js]: #### Tests:
```diff 
- 8 failing
```
```diff 
+ 14 passing
```
#### Test coverage: 
79.55 %

Comment (tcrosado) [inline test/AppCoinsIAB.js]: #### Tests:
```diff 
+ 22 passing
```
#### Test coverage: 
84.09 %

**Pull request merged**